keras/losses.py

```diff
@@ -1773,6 +1774,8 @@ def categorical_crossentropy(y_true,
   Returns:
     Categorical crossentropy loss value.
   """
+  #axis assert
+  check_ops.assert_integer_v2(int(axis), message=('`axis` must be of type `int`.'))
```

Comment:
Thank You, I have modified the changes.

Reply:
If I add `axis = int(axis)`, It will allow boolean values as well. In that case this change doesn't address the reported bug linked to this PR.
Made changes to address the boolean input in the axis.